src/switch_ivr_bridge.c

```diff
@@ -65,6 +67,53 @@ static void text_bridge_thread(switch_core_session_t *session, void *obj)
 
 	vh->up = 1;
 
+	if (!vh->session_a) {
+		if (!zstr(vh->session_a_uuid)) {
+			if ((vh->session_a = switch_core_session_locate(vh->session_a_uuid))) {
+				if (switch_core_session_read_lock(vh->session_a) != SWITCH_STATUS_SUCCESS) {
```

Comment:
beware that `switch_core_session_locate()` will obtain `read_lock` and return the session, no need to call `switch_core_session_read_lock()` yourself!